### PR TITLE
Added breast cancer classifier

### LINKED ISSUE #10: Create a Random Forest Classifier to predict Breast Cancer using Numerical Data

### PR DESCRIPTION
**Pull request by ckmganesh**
Fixes #10

Added the breast cancer classifier as described by the issue. The accuracy of the model can also be seen when the script is run

### CODE REVIEW

File: randomforest_classifier.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Y_pred = classifier.predict(X_test)
+print("Total observation :",Y_pred.shape[0] )
+#Confusion Matrix
+from sklearn.metrics import confusion_matrix
+cm = confusion_matrix(Y_test, Y_pred)
+print(cm)
+
+##Accuracy
+print("Accuracy :",(cm[0][0]+cm[1][1])/Y_pred.shape[0])
```
Comment (srimani-programmer) [inline randomforest_classifier.py:50]: @ckmganesh Add new line at the end.

**Review by srimani-programmer — CHANGES_REQUESTED** [review]
Please add your model inside models folder.

**Comment by srimani-programmer** [comment]
@ckmganesh please make your PR to New_Pipeline Branch.

**Comment by ckmganesh** [comment]
Check the other PR. Ignore this one. All the required changes are done on that PR. Please check it out

**Review by srimani-programmer — CHANGES_REQUESTED** [review]
Something went worng with this PR. Please make a new PR. It is saying unknown PR.

**Comment by ckmganesh** [comment]
I've created another PR. I already mentioned it before. Check #34 .
I'm closing this PR
